Bottom: 3b49c589c7eb0924305bfeaea7e6128f4674e746
Top:    d923e46fe2fc4bc1a982272bd59f8b7db2bb88f4
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-24 20:56:23 +0800

Refresh of virtio_ring-enable-event-idx

---

diff --git a/drivers/virtio/virtio_ring.c b/drivers/virtio/virtio_ring.c
index 5c3087f..ea85014 100644
--- a/drivers/virtio/virtio_ring.c
+++ b/drivers/virtio/virtio_ring.c
@@ -1232,10 +1232,9 @@ static unsigned virtqueue_enable_cb_prepare_packed(struct virtqueue *_vq)
 		// Setting VRING_EVENT_F_DESC in this function
 		// will break netperf test for now.
 		// Will need to look into this.
-#if 0
 		vq->event_flags_shadow = vq->event ? VRING_EVENT_F_DESC :
 						     VRING_EVENT_F_ENABLE;
-#else
+#if 0
 		vq->event_flags_shadow = VRING_EVENT_F_ENABLE;
 #endif
 		vq->vring_packed.driver->flags = cpu_to_virtio16(_vq->vdev,
